tools/validators/instance_validator/validate/telemetry_validator.py

```diff
@@ -22,12 +22,20 @@
 
 from validate import telemetry
 from validate import telemetry_error
+from validate import telemetry_warning
 
 DEVICE_ID = 'deviceId'
 
 class TelemetryValidator(object):
   """Validates telemetry messages against a building config file.
 
+  Attributes;
+    entities: a dict with entity_name as a key and EntityIntance as value.
+    timeout: the max time the validator must read messages from pubsub.
+    callback: the method called by the pubsub listener upon receiving a msg.
```

Comment:
Actually, this is the callback that the validator calls when it's done (either because messages for all entities were seen or because the timeout duration was reached).

Reply:
updated